tests/whitebox/_misc.py

```diff
@@ -67,11 +67,13 @@ def setup(self):
             raise RuntimeError(
                 "STRATISD environment variable must be set to absolute path of stratisd executable"
             )
-        self._stratisd = subprocess.Popen(  # pylint: disable=attribute-defined-outside-init
-            [os.path.join(stratisd_var), "--sim"],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-            universal_newlines=True,
+        self._stratisd = (
+            subprocess.Popen(  # pylint: disable=attribute-defined-outside-init
```

Comment:
Put this disable after the open paren on the line above and pylint will stop compalining.

Reply:
Resolved.